llama_stack/providers/remote/vector_io/milvus/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Dict
+
+from llama_stack.providers.datatypes import Api, ProviderSpec
+
+from .config import MilvusVectorIOConfig
+
+
+async def get_adapter_impl(config: MilvusVectorIOConfig, deps: Dict[Api, ProviderSpec]):
+    from .milvus import MilvusVectorIOAdapter
```

Comment:
I *think* the inline and remote implementations are equivalent.

If that is, indeed, the case can you follow the pattern that's done by Chroma? 

See here: https://github.com/meta-llama/llama-stack/blob/main/llama_stack/providers/inline/vector_io/chroma/__init__.py#L15

In short, they just import the inline implementation in the `__init__.py` and don't have an inline `chromadb.py`.